wiki/mapping/mediocre-map-assistant.md

```diff
@@ -64,7 +48,40 @@ instead of
 `C:\Steam\steamapps\common\Beat SaberBeat Saber_Data\CustomLevels`
 :::
 
-#### Check the WIP File Path
+### Linux Installation
+1. Download MMA2.zip from [GitHub](https://github.com/Assistant/MediocreMapAssistant2/releases/latest) 
+:::danger NOTE
+The download link for MMA2 is currently down. We are currently working to resolve this issue. Please hop over to the `#mapping-discussion` channel in the [BSMG Discord](https://discord.gg/beatsabermods) for more info. 
+:::
+
+2. Unzip the file and place the extracted folder wherever you like on your hard drive.
+3. Open Steam. At the top of the window, go to `Steam > Settings > Steam Play`. In the `Advanced` section, check `Enable Steam Play for all other titles` if it isn't already. Restart Steam if you are prompted to do so.
+3. On the bottom right of Steam's main window, click `Add a Game`, then `Add a Non-Steam Games`. A window opens, click `Browse` at the bottom. Select `All Files` as the file type. Find where `MediocreMapAssistant2.exe` is, and add it.
+4. Search for `Proton` in the Steam search bar, then install the latest version.
+5. Right-click the `MediocreMapAssistant2.exe` shortcut, and open the Properties menu. Check `Force the use of a specific Steam Play compatibility tool`, and select the latest version of Proton in the dropdown menu. At this step, you might also want to rename and add an icon to the shortcut as Linux cannot read the built in logo for MMA2.
+6. You can now use MMA2 by starting it through Steam. You can also create a shortcut by right-clicking the MMA2 shortcut in Steam, then under the `Manage` section clicking `Add Desktop Shortcut`.
+
+::: warning 
+It is **VERY** important to unzip the file in step 2 and only run the exe in the new folder instead of the one inside the zip as not doing so will cause issues with map creation and editing!
+:::
+
+#### Linux First Time Setup
+The first time you run MMA2 you will need to direct it to the location of two folders: `CustomLevels` and `CustomWIPLevels`. You have several options available to you:
+
+**If you have Beat Saber installed on the computer:**
+* Give MMA2 the path to the two appropriate folders
+* Not Known... Let us know what you find by filling out this [form](https://docs.google.com/forms/d/e/1FAIpQLSfVS6_EMZOujxthR3lTa2eEwHg5C3x1INouLgnbHhBDpv1M5A/viewform)!
+
+**If you have the Quest version -OR- don’t have Beat Saber installed on the computer:**
+* Make two folders called `CustomLevels` and `CustomWIPLevels` and then provide the path to them. Note that instead of the usual Linux path `/home/bsmg/apps/MMA2/CustomLevels` and `/home/bsmg/apps/MMA2/CustomWIPLevels`, they need to be prefixed with `Z:`.
```

Comment:
I wouldn't use a real path without saying "Example".

i'd rather use in this context `/path/to/CustomLevels` and `/path/to/CustomWIPLevels` and write their respective Proton-compatible equivalents (starting with `Z:`)

Reply:
I modified it now and it's good, it can now be merged (the "BeatSaber installed" part is not wrong, just kinda seems useless)